### ISSUE TITLE
McMMO potion saturation bug

### ISSUE THREAD

**Issue by MrXmarks** [mcMMO-Dev/mcMMO#3115]
Potion saturation from mcmmo does not work.

**Comment by ProgrammingAtDawn** [comment]
I have the same issue. Neither normal potion nor modified ones with redstone/glowstone worked.

Started looking through the code and found this workaround for splash potion of saturation:
https://github.com/mcMMO-Dev/mcMMO/blob/bb27d753b1fb0217e30cfbe6bb243014b057456a/src/main/java/com/gmail/nossr50/listeners/EntityListener.java#L688

So I brewed a splash potion and it worked. With redstone/glowstone modifier and without.

Seems like the workaround needs also be implemented for the normal non-throwable potions of saturation.

**Comment by sigmaroot** [comment]
Confirmed for me too.

**Comment by sigmaroot** [comment]
Still not fixed...

**Comment by Glitchfinder** [comment]
Is this still an issue you're having? If so, is it resolved by [the latest dev build](https://popicraft.net/jenkins/job/mcMMO/) for mcMMO or the latest build of spigot/bukkit?

**Comment by sigmaroot** [comment]
Yes, not fixed. Tested with b16.

**Comment by sigmaroot** [comment]
Is this fixed or just closed without any further attention?

**Comment by t00thpick1** [comment]
I did a general purge of tickets likely to be made outdated by the 1.13
update I'm working on as the code will be changed dramatically in some
areas.  If the issue still exists after the full 1.13 release file a new
ticket.  But the tracker was full of unconfirmed or old bugs that some or
most of got fixed since, and the rest of will be affected by the 1.13
update.  Leaving them open was just making tracking things more difficult.
